install.sh

```diff
@@ -8,7 +8,7 @@ apt-get install -y libxml2 libxml2-dev libxslt1.1 libxslt1-dev libffi-dev libssl
 # Installing pip-9.0.1
 wget -nv https://bootstrap.pypa.io/3.2/get-pip.py
 python get-pip.py
-python -m pip install pip==8.1.2
+pip install -U pip
```

Comment:
can we not specify exact version. thisl will alway upgrade it to latest

Reply:
if we specified the version also it will take latest only and also it will show in logs like pip upgraded version is available .

![image](https://user-images.githubusercontent.com/37403043/44076594-64c4ce6a-9fbe-11e8-9e94-ad2e52ce5d8d.png)

Reply:
thats ok. we need to specify the version even if it takes it by default. this is because we want the script to install the same version whenever we run it. if we dont do this, everytime someone runs this, they'll get a new version

Reply:
okay i will specified the version as per launchpad https://launchpad.net/ubuntu/xenial/+source/python-pip